packages/webdriver/src/utils.ts

```diff
@@ -226,11 +226,18 @@ export function getErrorFromResponseBody (body: any) {
 export class CustomRequestError extends Error {
     constructor(body: WebDriverResponse) {
         const errorObj = body.value || body
-        super(errorObj.message || errorObj.class || 'unknown error')
+        const name = errorObj.name || 'WebDriver Error'
+        super(name + ': ' +(errorObj.message || errorObj.class || 'unknown error'))
```

Comment:
I think the following is better readable
```suggestion
        super(`${name}: ${errorObj.message || errorObj.class || 'unknown error'}`)
```